src/Registration/redux/userReducer.tsx

```diff
@@ -0,0 +1,23 @@
+import { User } from '../User';
+import { ADD_USER, AddUserAction } from './userActions';
+
+export type UsersState = {
+  users: User[];
+};
+
+const initialState: UsersState = {
+  users: [],
+};
+
+export function usersReducer(
```

Comment:
Why the decision not to use `redux-actions`' `handleActions`, seeing as you're already using `createAction` from the same library?